src/features/import/utils/problems/predictProblems.spec.ts

```diff
@@ -459,6 +485,65 @@ describe('predictProblem()', () => {
     ]);
   });
 
+  it('returns problem when either the first name or last name value is missing', () => {
+    const sheet = makeFullSheet({
+      columns: [
+        {
+          field: 'first_name',
+          kind: ColumnKind.FIELD,
+          selected: true,
+        },
+        {
+          field: 'last_name',
+          kind: ColumnKind.FIELD,
+          selected: true,
+        },
+        {
+          field: 'email',
+          kind: ColumnKind.FIELD,
+          selected: true,
+        },
+      ],
+      rows: [
+        { data: ['Clara', 'Zetkin', 'zetkin@example.com'] },
+        { data: ['John', '', 'john@example.com'] },
+      ],
+    });
+
+    const problems = predictProblems(sheet, 'SE', []);
+    expect(problems).toEqual([
+      {
+        indices: [1],
+        kind: ImportProblemKind.MISSING_ID_AND_NAME,
+      },
+    ]);
+  });
+
+  it('returns correct problem when column is not configured', () => {
+    const sheet = makeFullSheet({
+      columns: [
+        {
+          kind: ColumnKind.UNKNOWN,
+          selected: false,
+        },
+      ],
+      rows: [
+        { data: ['Clara', 'Zetkin', 'zetkin@example.com'] },
+        { data: ['John', '', 'john@example.com'] },
+      ],
+    });
+
+    const problems = predictProblems(sheet, 'SE', []);
+    expect(problems).toEqual([
+      {
+        kind: ImportProblemKind.UNCONFIGURED_ID_AND_NAME,
+      },
+      {
+        kind: 'NO_IMPACT',
```

Comment:
I added a new test where column is not configured. And I got this 'NO_IMPACT' together in problem result. Is this fine?